icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/controller/ProductController.java

```diff
@@ -38,4 +39,16 @@ public ResponseEntity<List<ProductDTO>> getProducts() {
                 .map(product -> modelMapper.map(product, ProductDTO.class))
                 .collect(Collectors.toList()));
     }
+
+    @PostMapping("/products")
```

Comment:
As dori sa te mai gandesti putin la cum denumesti acest endpoint. In cazul tau, ai deja un`@RequestMapping(path = "/products")` pe controller, ceea ce in final va determina endpoint-ul tau sa se apeleze prin POST `URL/products/products` , ceea ce nu e foarte relevant pentru ce face endpoint-ul. Incearca sa redenumesti putin aici sa fie cat mai relevant pentru ce faci cu acest endpoint

Reply:
aici am denumit asa pentru ca asa scrie in tichet

Reply:
In tichet din cate vad trebuie sa apelezi endpoint-ul POST `/products`. Aceasta cerinta este deja indeplinita pentru ca ai acel RequestMapping pe controller. Tu prin adaugarea inca unui `PostMapping("/products")` pe metoda ta din controller, transformi endpoint-ul din `/products` in `/products/products`. Adnotarea de pe controller va prefixa toate URL-urile de pe metodele aflate in el. Sfatul meu ar fi sa stergi `"/products"` din PostMapping-ul de pe metoda ta